benchmarks/graphics_pipeline/main.cpp

```diff
@@ -663,11 +715,17 @@ void ProjApp::LoadNodes(
     }
 }
 
+void ProjApp::SetupCamera()
+{
+    mCamera.LookAt(mCamera.GetEyePosition(), mCamera.GetTarget());
```

Comment:
Why not call these in the constructor? It sounds like these have to be called, otherwise the camera is in the invalid state?